src/commands/common/asset-sync-command.js

```diff
@@ -110,13 +97,6 @@ class AssetSyncCommand extends Command {
                     stateIndex,
                 )
             ) {
-                await this.repositoryModuleManager.updateAssetSyncRecord(
```

Comment:
Just to check, why did we have this before?

Reply:
I am not sure. I think it doesn't make sense. I reviewed with @djordjekovac and we decided to change the logic there

Reply:
This was an error in older code, it doesn't make sense to be there